<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_second">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>전송</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTxt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>401</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainMessage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>481</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_disconnect">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>연결 끊기</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_ipaddr">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblIPAddress">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_port">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPort">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_testConnect">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>연결</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPasswd">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passwd:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmail">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_passwd">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_email">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_mariadb_ip_addr">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblMariaDB_IpAddr">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MariaDB IP Addr</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
